data/core/src/commonMain/kotlin/com/tunjid/heron/data/repository/PostRepository.kt

```diff
@@ -146,6 +149,7 @@ internal class OfflinePostRepository @Inject constructor(
     private val transactionWriter: TransactionWriter,
     private val fileManager: FileManager,
     private val savedStateDataSource: SavedStateDataSource,
+    private val timelineRepository: TimelineRepository,
```

Comment:
Repositories depending on repositories can lead to circular dependencies. Its safer to depend on the labelDao instead, and use it to read the labels. Its fine if we don't refresh the labels from the network here.